src/components/Modal/index.js

```diff
@@ -68,6 +71,20 @@ export function Modal({
                   style={closeButtonStyle}
                   className={`pointer-events-none sm:pointer-events-auto absolute top-0 right-0 ${closeButtonClass}`}
                 >
+                  {dropDownVisible && (
```

Comment:
Why are we adding this?

Reply:
<img width="259" alt="Screenshot 2022-02-04 at 12 06 23 PM" src="https://user-images.githubusercontent.com/32077311/152510679-b5e8b838-e359-490a-93e1-b4dc8a80aed0.png">
it is dropdown which besides close button

Reply:
Yes, but there is no need for this dropdpown to be added by default to every modal